common/configurable/src/main/java/io/helidon/common/configurable/ResourceImpl.java

```diff
@@ -113,6 +120,12 @@ public void cacheBytes() {
         }
     }
 
+    @Override
+    public String toString() {
```

Comment:
why do we still need a custom toString() for this?

Reply:
Yes, this is not a generated type, and the configuration is translated into things that are not available on it